src/pages/profile/index.ts

```diff
@@ -0,0 +1,13 @@
+import { GetServerSideProps } from 'next'
+import ProfilePage from 'components/auth/profile/ProfilePage'
+
+export const getServerSideProps: GetServerSideProps = async () => {
+  return {
+    redirect: {
+      destination: '/profile/donations',
+      permanent: false,
+    },
+  }
+}
```

Comment:
You can achieve that by using `next.config.js` rewrites 

Read more at https://nextjs.org/docs/api-reference/next.config.js/rewrites

Reply:
We also kind of miss the secured props here, no?

Reply:
Done in https://github.com/podkrepi-bg/frontend/pull/703/commits/fb6522f27f263b17ebc1af24cceadf7dc4af0b66